cpu/cortexm_common/include/cpu.h

```diff
@@ -114,6 +187,26 @@ static inline void cortexm_isr_end(void)
     }
 }
 
+/**
+ *  @brief   Branch the execution to a specified address, usually used to
+ *           jump to another RIOT instance.
+ */
+static inline void cortexm_branch_address(uint32_t destination_address)
```

Comment:
Yeah I had it too, apparently that option is only valid for M3 mcu.

I have a branch with some modifications to work with samr21 but there's something not working with the metadata.